test/provider/expect.unittest.ts

```diff
@@ -51,7 +51,7 @@ type TestParam = [
 const FAKE_TEST_DURATION = 10000;
 
 const REGISTRATION_TIMEOUT = 5000;
-const PENDING_TIMEOUT = 250;
+const PENDING_TIMEOUT = 100;
```

Comment:
This should be the same value as `Timeouts.WINDOW_EXPECT_TO_SEEN` - read as "how soon after the window is expected should the promise reject, if the window is never created".

I didn't simply import Timeouts originally, as then I felt like I was leaning on part of implementation of the thing I was testing, but maybe it should just be imported 🤷‍♀️. Or just a comment here?